src/main/kotlin/com/fasterxml/jackson/module/kotlin/KotlinModule.kt

```diff
@@ -14,16 +14,22 @@ class KotlinModule constructor (
     val reflectionCacheSize: Int = 512,
     val nullToEmptyCollection: Boolean = false,
     val nullToEmptyMap: Boolean = false,
-    val nullisSameAsDefault: Boolean = false
+    val nullIsSameAsDefault: Boolean = false
```

Comment:
I fixed the camelCasing on this property.

Reply:
good, thanks.